src/containers/App.jsx

```diff
@@ -139,7 +139,7 @@ const App = ({
           )}
           {isReady && (
             <Switch>
-              <Route path="/redirect" component={IntentRedirect} />
+              <Route path="/redirect" render={() => <IntentRedirect />} />
```

Comment:
Good catch I think you're right we have to pass the props.
Also this was done because react-router was complaining the IntentRedirect was not a valid component props (I guess because of the HoC?).